apps/client/src/initialize-auth-dependents.tsx

```diff
@@ -0,0 +1,18 @@
+import { v4 as uuid } from 'uuid';
+import { utcDateString } from './helpers/dates';
+import { cloudWatchMetricsRecorder } from './metrics/cloud-watch-metrics-recorder';
+import { cloudWatchLogger } from './logging/cloud-watch-logger';
+
+/**
+ * This function initializes the services that depends on authentication to work
+ * @param applicationName the name of this application
+ */
+export async function initializeAuthDependents(applicationName: string) {
```

Comment:
use function expression

Reply:
there's not a convention to use function expression for all function in iot-application
imo, there's very little value to establish this convention
do you think we should establish the convention? use the lint rule? https://eslint.org/docs/latest/rules/func-style